sklearn/metrics/tests/test_score_objects.py

```diff
@@ -37,18 +41,20 @@
 from sklearn.externals import joblib
 
 
-REGRESSION_SCORERS = ['r2', 'mean_absolute_error', 'mean_squared_error',
-                      'median_absolute_error']
+REGRESSION_SCORERS = ('r2', 'mean_absolute_error', 'mean_squared_error',
+                      'median_absolute_error')
 
-CLF_SCORERS = ['accuracy', 'f1', 'f1_weighted', 'f1_macro', 'f1_micro',
+CLF_SCORERS = ('accuracy', 'f1', 'f1_weighted', 'f1_macro', 'f1_micro',
                'roc_auc', 'average_precision', 'precision',
                'precision_weighted', 'precision_macro', 'precision_micro',
                'recall', 'recall_weighted', 'recall_macro', 'recall_micro',
                'log_loss',
                'adjusted_rand_score'  # not really, but works
-               ]
+               )
```

Comment:
[`brier_score_loss`](http://scikit-learn.org/dev/modules/generated/sklearn.metrics.brier_score_loss.html#sklearn.metrics.brier_score_loss) and [`cohen_kappa_score`](http://scikit-learn.org/dev/modules/generated/sklearn.metrics.cohen_kappa_score.html) are missing here.